http/server.lua

```diff
@@ -1,6 +1,7 @@
 -- http.server
 
 local lib = require('http.lib')
+local _, sslsocket = pcall(require, 'http.sslsocket')
```

Comment:
It would be nice to check the support before usage.
```suggestion
local sslsocket_supported, sslsocket = pcall(require, 'http.sslsocket')
```

Reply:
Added check on this.